package.json

```diff
@@ -39,40 +39,40 @@
     "pages": "npm run storybook:build && cp -r storybook-static _site"
   },
   "dependencies": {
-    "@uswds/uswds": "^3.8.2",
-    "lit": "^3.2.0",
-    "sass": "^1.78.0"
+    "@uswds/uswds": "^3.9.0",
+    "lit": "^3.2.1",
+    "sass": "^1.79.4"
```

Comment:
## Sass deprecation warning

This update brings on a new Sass deprecation warning

>[!WARNING]
> DEPRECATION WARNING: The legacy JS API is deprecated and will be removed in Dart Sass 2.0.0.
[More info →](https://sass-lang.com/documentation/breaking-changes/legacy-js-api/)
> 

In the `bundlers` section of the above guidance there is specific instructions for Vite.

> Vite still defaults to the legacy API, but you can similarly switch it by setting `api` to `"modern"` or `"modern-compiler"`. See [Vite’s documentation](https://vitejs.dev/config/shared-options.html#css-preprocessoroptions)
> 

Currently, switching to the modern api causes an error. I *think* that this is due to USWDS using the deprecated sass `render()` function.

I've created #77 to track. I’ll readdress after completing uswds/uswds#6103.